modules/table/src/main/java/org/apache/ignite/internal/table/distributed/StorageUpdateHandler.java

```diff
@@ -195,11 +193,11 @@ public void handleUpdateAll(
     void executeBatchGc() {
         HybridTimestamp lwm = lowWatermark.getLowWatermark();
 
-        if (lwm == null || safeTimeTracker.current().compareTo(lwm) < 0) {
+        if (lwm == null || gcUpdateHandler.getSafeTimeTracker().current().compareTo(lwm) < 0) {
```

Comment:
How about extracting the check to a method on GcUpdateHandler? It could be called `safeTimeBelow(HybridTimestamp lwm)` or similarly

Reply:
If you insist, I can add, but at the moment it seems that for a single method it is a bit redundant.

Reply:
It's up to you